### PR TITLE
Hide state diff if failed tx

### PR DESCRIPTION
**Pull request by gesseekur**

### CODE REVIEW

File: newclient/src/apollo/schemas/api.json [hunk 1]
```diff
@@ -1661,7 +1661,7 @@
             "deprecationReason": null
           },
           {
-            "name": "price_change_percentage_24h",
+            "name": "price_change_24h",
```
Comment (olgakup) [inline newclient/src/apollo/schemas/api.json]: hey why was this changed?

**Review by olgakup — CHANGES_REQUESTED** [review]

File: newclient/src/apollo/schemas/api.json (hunk 1 above)
Comment (gesseekur) [reply newclient/src/apollo/schemas/api.json]: @olgakup hrmmm this must have been mixed up with some other file cause I don't think i touched this, do I just go ahead and change it directly on this file? getLatestPrices.ts still seems to have the correct attribute

**Review by olgakup — APPROVED** [review]
